src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -107,30 +107,21 @@ public override DbCommand GetCommand(DbConnection connection)
         /// </summary>
         private bool CheckForDuplicateDeleteRows(WhereClause where, string input, DbConnection connection)
         {
+            int count = 0;
```

Comment:
This is unnecessary - just make line 117 

`return Convert.ToInt32(command.ExecuteScalar()) <= 1`

I would also suggest 

1. Renaming the method to HasDuplicateRows (more descriptive name)
1. Change the logic to check for `>1`
1. Change the check on line 92 to be `if(HasDuplicateRows(...))`